File: 05--Resources/UnwindSegueApp/UnwindSegueApp/ViewController.m
//
//  ViewController.m
//  UnwindSegueApp
//
//  Created by Phil Wright on 5/27/16.
//  Copyright © 2016 Touchopia, LLC. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    
}

- (IBAction)unwindSegue:(UIStoryboardSegue *)segue {
    
    NSLog(@"unwindSegue");
    
    [self reset];
    
}


- (IBAction)travelBackTapped:(UIButton *)sender {
    
    [self performSegueWithIdentifier:@"PopupSegue" sender:nil];
}

-(void)reset {
    NSLog(@"Reset");
    
    // set milesPerHour = 0
    // set
}


@end

File: 23-Forecaster/NavApp/NavApp/ViewController.swift
//
//  ViewController.swift
//  NavApp
//
//  Created by Phil Wright on 6/23/16.
//  Copyright © 2016 Touchopia, LLC. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    var currentCity = City()
    
    override func viewDidLoad() {
        super.viewDidLoad()
       
        self.title = "Maps"
        
    }

    @IBAction func MyUnwindSegue(segue: UIStoryboardSegue) {
    
        if let cityController = segue.sourceViewController as? CityViewController {
            
            currentCity.name = cityController.cityString
            currentCity.zipCode = cityController.zipCodeString
            
            print(cityController.cityString)
            print(cityController.zipCodeString)
            
        }
        
        
        print("Unwind called")
        
    }

    

}

